assignments/arrays.js

```diff
@@ -63,34 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make}  ${inventory[32].car_model}` );
 
 
 
 // ==== Challenge 2 ====
```

Comment:
While your answer is correct for this static data set, it will not work if we ever change or update the data set. So your answer isn't dynamic for the question being asked. 

Go back and try to re-answer this one with the mindset that the last item in the array could be changed and you'd still want your function to be correct.

Reply:
fixed